zephyr/projects/zxa_example_project/src/main.c

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Please move this to `<root>/app`

Reply:
See reasons above for why this *shouldn't* be moved to `<root>/app`.